tests/acceptance/src/test/resources/org/activityinfo/test/acceptance/design.feature

```diff
@@ -0,0 +1,12 @@
+@web
```

Comment:
This could be annotated with "@api" as well as it would be a good test of the REST api.